e2e/test/scenarios/embedding/embedding-smoketests.cy.spec.js

```diff
@@ -58,18 +58,7 @@ describe("scenarios > embedding > smoke tests", { tags: "@OSS" }, () => {
       });
 
       cy.location("pathname").should("eq", embeddingPage);
-      cy.findByTestId("enable-embedding-setting").within(() => {
-        cy.findByText(embeddingDescription);
-
-        cy.findByLabelText("Enable Embedding").click({ force: true });
-      });
-      // The URL should stay the same
-      cy.location("pathname").should("eq", embeddingPage);
-
-      cy.findByTestId("enable-embedding-setting").within(() => {
-        cy.findByRole("checkbox").should("be.checked");
-      });
-
+      mainPage().findByText(embeddingDescription).should("be.visible");
```

Comment:
move the toggle assertion below to test if other elements update when the static embed toggle is flipped.